src/lib/stories.ts

```diff
@@ -319,6 +319,8 @@ const printFormattedStory = (program: any) => {
                 )
                 .replace(/%u/, url)
                 .replace(/%a/, `${story.archived}`)
+                .replace(/%gbf/, `${buildStoryBranch(story)}`)
+                .replace(/%gbs/, `${buildStoryBranch(story, `${config.mentionName}/ch${story.id}/`)}`)
```

Comment:
I had to slightly change the `gb` replacement to be `gbf` in order for it not to conflict with the new git branch short `gbs`. This enables both full branch display and short branch display using the prefix argument for buildStoryBranch

Reply:
What if we go with `gb` for the default/"full" branch and `gbs` for the short/"slug" version? That would be possible as long as the `replace` command for `gbs` is applied before that for `gb` (Similar to how `updated` is applied before `u` for the URL formatter).

Reply:
Ah, clever. We can do that